### PR TITLE
Update Rounded Buttons. Issue #34

### PR DESCRIPTION
**Pull request by stellasu168**
@regnerjr Hey John, please take a look and merge me. Thank you!

### CODE REVIEW

File: Connnect/JoinViewController.swift [hunk 1]
```diff
@@ -16,10 +16,7 @@ class JoinViewController: UIViewController {
     override func viewDidLoad() {
```
Comment (regnerjr) [inline Connnect/JoinViewController.swift]: Now that there is no code here anymore we could lose the `viewDidLoad` function too.

File: Connnect/NewChannelViewController.swift [hunk 2]
```diff
@@ -16,10 +16,6 @@ class NewChannelViewController: UIViewController {
     override func viewDidLoad() {
```
Comment (regnerjr) [inline Connnect/NewChannelViewController.swift]: Empty `viewDidLoad` here too.

File: CustomUIButton.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//
+//  CustomUIButton.swift
+//  Connnect
+//
+//  Created by Stella Su on 11/22/16.
+//  Copyright © 2016 iOS-Connect. All rights reserved.
+//
+
+import Foundation
```
Comment (regnerjr) [inline CustomUIButton.swift]: `import UIKit` pulls in `Foundation` as well so we could leave off this `import Foundation`

File: CustomUIButton.swift [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import UIKit
+
+class CustomUIButton: UIButton{
```
Comment (regnerjr) [inline CustomUIButton.swift]: This looks super 👍 🎉 👍 💯 

My only suggestion would be to give it a more descriptive name. Maybe something like `BorderedButton` or `RoundedButton` or maybe just `Button`

File: CustomUIButton.swift [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.backgroundColor = .clear
+        self.layer.cornerRadius = 10
+        self.layer.borderWidth = 2
+        
+        self.layer.borderColor = self.tintColor.cgColor
+
+    }
+
```
Comment (regnerjr) [inline CustomUIButton.swift]: Then to be really thorough we should also override the convenience initializer for UIButton, so users of this class always get the button they expect. 

I thing the initializer I'm thinking of is something like `init(buttonStyle: UIButtonStyle)`

**Review by regnerjr — APPROVED** [review]
Hey friend these changes look great. I wanted to give you all the feedback that I possibly could in hopes that it would help you. Feel free to merge if you think this is good enough or whatever. Nice work 💯

**Comment by stellasu168** [comment]
Thank you for your feedback @regnerjr ! It's super helpful. I can't figure out how to override the initializer for UIButton. I've tried your suggestion init(buttonStyle: UIButtonStyle) but ran into errors. Any more suggestions? Or we can talk about it later. Thank you!

**Review by regnerjr — COMMENTED** [review]
This looks great

**Comment by regnerjr** [comment]
Looks like I might have gotten the name of that function wrong. 

https://developer.apple.com/reference/uikit/uibutton/1624028-init

Does this help.

**Pull request merged**